Projects/Embark/Sources/Embark.swift

```diff
@@ -184,6 +184,33 @@ extension Embark: Presentable {
             }
 
             viewController.navigationItem.leftBarButtonItem = backButton
+            
+            let tooltipButton = UIButton()
+            tooltipButton.isHidden = true
+            tooltipButton.setImage(hCoreUIAssets.infoLarge.image, for: .normal)
+            
+            bag += state.passageTooltipSignal.atOnce().onValue({ (shouldShow) in
+                tooltipButton.isHidden = !shouldShow
```

Comment:
Unsure what is the best practice here, but couldn't we use `        viewController.navigationItem.setRightBarButtonItems([], animated: true)`?